apm-agent-plugins/apm-apache-httpclient/apm-apache-httpclient-common/pom.xml

```diff
@@ -0,0 +1,36 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<project xmlns="http://maven.apache.org/POM/4.0.0"
+         xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+         xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
+    <modelVersion>4.0.0</modelVersion>
+
+    <parent>
+        <artifactId>apm-apache-httpclient</artifactId>
+        <groupId>co.elastic.apm</groupId>
+        <version>1.43.1-SNAPSHOT</version>
+    </parent>
+
+    <artifactId>apm-apache-httpclient-common</artifactId>
+    <name>${project.groupId}:${project.artifactId}</name>
+
+    <properties>
+        <apm-agent-parent.base.dir>${project.basedir}/../../..</apm-agent-parent.base.dir>
+    </properties>
+
+
+    <build>
```

Comment:
This project doesn't expose any shared tests, so no need to expose a test-jar. Please remove the `<plugins>` section.

Reply:
fixed